Functions/CatalogItems/Get-RsItemReference.ps1

```diff
@@ -0,0 +1,111 @@
+# Copyright (c) 2016 Microsoft Corporation. All Rights Reserved.
+# Licensed under the MIT License (MIT)
+
+
+function Get-RsItemReference
```

Comment:
Because the official PowerShell style guidance requires the noun in the <Verb>-<Noun> naming convention of PowerShell to be singular.